cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go

```diff
@@ -52,41 +76,38 @@ limitations under the License.
 //	  node-cidr-mask-size: 20
 //	schedulerExtraArgs:
 //	  address: "10.100.0.1"
+//  apiServerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
+//  controllerManagerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
+//  schedulerExtraVolumes:
+//  - name: "some-volume"
+//    hostPath: "/etc/some-path"
+//    mountPath: "/etc/some-pod-path"
+//    writable: true
+//    pathType: File
 //	apiServerCertSANs:
```

Comment:
this sort of looks like it should be indented one level less?

Reply:
Actually this whole section's indentation is off

Reply:
yes, it totally is, good 👀